common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -573,7 +573,7 @@ def available_translations(self, transcripts, verify_assets=True):
 
         # If we're not verifying the assets, we just trust our field values
         if not verify_assets:
-            translations = list(other_lang)
+            translations = list(other_lang) if other_lang else list()
```

Comment:
What is the type of `other_lang` here?  The variable name makes it look like it's a single language, and if that object is a string, this could lead to unfortunate results:

    >>> list("en-US")
    ['e', 'n', '-', 'U', 'S']

If `other_lang` is already a list or iterator, it could use a better name to communicate that fact.